test/Microsoft.AspNetCore.Http.Abstractions.Tests/RunExtensionsTests.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder.Internal;
+using Microsoft.AspNetCore.Http;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Builder.Extensions
+{
+    public class RunExtensionsTests
+    {
+        [Fact]
+        public void Run_NullArguments_ArgumentNullException()
+        {
+            var builder = CreateBuilder();
+
+            Action nullConfiguration = () => builder.Run(null);
```

Comment:
var instead of Action? Or just place the action in the Assert.Throws statement, it's pretty clear what it's doing.